locales/pot/github.po

```diff
@@ -0,0 +1,22 @@
+# SOME DESCRIPTIVE TITLE.
+# Copyright (C) YEAR THE PACKAGE'S COPYRIGHT HOLDER
+# This file is distributed under the same license as the renovate-platform package.
+# FIRST AUTHOR <EMAIL@ADDRESS>, YEAR.
+#
+#, fuzzy
+msgid ""
+msgstr ""
+"Project-Id-Version: renovate-platform\n"
```

Comment:
I don't think we should bother with reviewing translations in a language we don't speak at a near-native level. We can't rely on Google Translate giving us correct translations, that are fit for purpose.

Can we decouple the translations from the main English Renovate project?

The Pro Git 2 book project has a main English language repository, and different repositories for each translation. Would something like that work for us?

- [Pro Git 2 book in English](https://github.com/progit/progit2)
- [Pro Git 2 book in German](https://github.com/progit/progit2-de)
- [Pro Git organization repository overview](https://github.com/progit)

Reply:
The basic setup for the translation should probably live in the English language main repository, so `renovatebot/renovate`. The translation files could then go in a new repository for each language.

Each translation would then be a fork of the main `renovatebot/renovate` repository.